app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
+  display: block;
+  background-color: $grey-medium;
+  padding: $gutter;
+  margin-top: $gutter * 2;
+}
+
+.writing-field {
+  display: inline-block;
+  margin-bottom: 24px;
+
+  label {
+    display: block;
+    margin-bottom: $gutter * 1.5;
+    margin-top: $gutter;
+  }
+
+  .space-for-writing {
+    border: 0;
+    border-bottom: 1px solid black;
+  }
+
+  &.name {
+    width: 100%;
+
+    .space-for-writing {
+      width: 97%;
+    }
+  }
+
+  &.signature, &.date {
```

Comment:
> On the one hand I'd say find a simple grid helper/thing and use that or come up with a name like .row and col-50

As far as I've seen from #ui-kit we're working on our own grid as part of elements. I don't want to assign generic names to these because I don't want to:

1. give the idea to devs coming after (by example) that this is in any way *the* grid system. I *want* devs to look at this and say "oh, ok, single use case".
2. design a grid system, even in miniature. My CSS is passably OK, but there are people around here better qualified 😉